backend/open_webui/models/memories.py

```
@@ -79,31 +79,31 @@ def update_memory_by_id(
                 return None
 
     def get_memories(self) -> list[MemoryModel]:
-        with get_db() as db:
+        with session_scope() as db:
             try:
                 memories = db.query(Memory).all()
                 return [MemoryModel.model_validate(memory) for memory in memories]
             except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)